src/op_mode/nat-translation.py

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env python3
+
+'''
+what we do and why
+'''
+
+import os
+import sys
+import ipaddress
+import argparse
+import xmltodict
+
+from vyos.util import popen
+from vyos.util import DEVNULL
+
+conntrack = '/usr/sbin/conntrack'
+
+verbose_format = "%-20s %-18s %-20s %-18s"
```

Comment:
There's a great library called tabulate. We use it in op_mode/show_dhcp.py for example.
Makes things so much simpler than manual formats, has built-in support for printing headers too.

Reply:
Yes, I have used it for other tools. In that case the output for "--verbose" cause every odd/even line to be formated differently so it is not really suitable.